tests/rptest/tests/partition_movement_test.py

```diff
@@ -153,15 +153,15 @@ def status_done():
             return converged and info["status"] == "done"
 
         # wait until redpanda reports complete
-        wait_until(status_done, timeout_sec=60, backoff_sec=2)
```

Comment:
Currently test_dynamic is only doing partition movements within nodes (i.e. there's no inter-node recovery traffic), movements should be pretty quick -- I think I measured these at about 5 seconds on average.

I'm okay with making this change to get the test back into operation, but we should come back to this and investigate slow executions some time.  It could be hiding a raft issue where maybe elections aren't proceeding as promptly as they should.